data/videos.yml

```diff
@@ -3693,3 +3693,30 @@ Seattle Xcoders February 2016:
     title: Continuous Integrations for iOS Apps
     tags: [xcode, build, continuous integration]
     vimeo: 158290425
+NSLondon February 2016:
+  - language: English
+    speakers: [Raimon Lapuente, Gergely Orosz]
+    title: "Sorry Objc: It’s not you, it’s me"
```

Comment:
I had a little look but I am not familiar with the Ruby/middleman debugging tools, or really any part of the stack. I made https://github.com/chriseidhof/pomotv/issues/112 to track this. I suppose there’s no pressing need to fix it now. I’m not going to look into this further at the moment.

For now, I used straight quotes for the title causing the problem and rebased onto master, so we can get these videos up. 😄